get-webpack-config.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-env node */
 const path = require('path');
 const webpack = require('webpack');
-const { isDebug, buildDir } = require('./build-arguments');
+const { isDebug, buildDir, isVrt } = require('./build-arguments');
```

Comment:
@langolf - instead of isVRt, let's make a new construction:

```js
const isInlined = isCompiling(buildDir);
```

Reply:
Good idea. We can avoid extra flag.